### PR TITLE
Update Compute docu content

### PR DESCRIPTION
**Pull request by dschwabeS11**
-  Better TOC
-  Cleaner page
-  Updated Debian Codename
-  Fix Image Upload commands

### CODE REVIEW

File: english/docs/documentation/compute.md [hunk 1]
```diff
@@ -68,11 +71,14 @@ This security group can now be connected to a port of your network:
 
 The security group "default" is added in this example, since this group is taking care of allowing outbound traffic.
 
-### Can I use my own images in the Compute Service?
-You can easily upload your own image and use it right after, like this:
+## Can I use my own images in the Compute Service?
+
+First download the image that you want to create
+```shell
+curl -O -J -L http://cdimage.debian.org/cdimage/openstack/testing/debian-testing-openstack-amd64.qcow2 -o debian-testing-openstack-amd64.qcow2
```
[the remark this answers is not in the document]
Comment (jpeschkeSys11) [reply english/docs/documentation/compute.md]: I see it's an open bug for now: https://bugzilla.redhat.com/show_bug.cgi?id=1493867 
But then we should mention that in the documentation and recommend using heat or something else

File: german/docs/documentation/compute.md [hunk 2]
```diff
@@ -16,19 +18,19 @@ Aktuell bieten wir 4 verschiedene Instanz Typen.
...
+M1 Large  |  m1.large   |  32GB  |   8   |   50GB   | hoch
 
-*Jeder Instanz kann via unserem Block Storage Service weiterer Speicher hinzugefügt werden.
+* Jeder Instanz kann via unserem Block Storage Service weiterer Speicher hinzugefügt werden.
 
-## FAQ
+# FAQ
 
-### Kann ich einer Compute Instanz eine feste interne IP zuweisen?
+## Kann ich einer Compute Instanz eine feste interne IP zuweisen?
```
[the remark this answers is not in the document]
Comment (dschwabeS11) [reply german/docs/documentation/compute.md]: This was a WIP branch. Sorry for that. Will watch out for this in the future. Can we merge this for now as I'll rework the whole compute documentation anyways soon?

File: german/docs/documentation/compute.md [hunk 3]
```diff
@@ -94,28 +96,38 @@ Im sich nun öffnenden Fenster haben wir verschiedene Eingabe-Optionen: <br>
  
 Sind nun alle Felder ausgefüllt, klicken wir auf "Create Image" und laden unser eigenes Image in den SysEleven Stack
  
-#### OpenStack-CLI
+### OpenStack-CLI
 Hierfür ist es notwendig [OpenStack-CLI](http://docs.openstack.org/user-guide/common/cli-install-openstack-command-line-clients.html) installiert zu haben.
-Nun können wir mit unserer [gesourcten openrc.sh](https://doc.syselevenstack.com/tutorials/02-kickstart/#zugriff-auf-die-syseleven-stack-api) unser Image hochladen und zwar mit folgendem Befehl:
+Nun können wir mit unserer [gesourcten openrc.sh](https://doc.syselevenstack.com/tutorials/02-kickstart/#zugriff-auf-die-syseleven-stack-api) unser Image hochladen und zwar mit folgenden Befehlen.
+
+Zuerst laden wir uns das Image herunter:
```
[the remark this answers is not in the document]
Comment (dschwabeS11) [reply german/docs/documentation/compute.md]: Which "above" exactly?

File: english/docs/documentation/compute.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dschwabeS11) [reply english/docs/documentation/compute.md]: Added

[the remark this answers is not in the document]
Comment (dschwabeS11) [reply english/docs/documentation/compute.md]: Adjusted.

File: german/docs/documentation/compute.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (dschwabeS11) [reply german/docs/documentation/compute.md]: Done

**Pull request closed without merging**